### PR TITLE
Mohsen / reorganized deps and dev deps

### PR DESCRIPTION
**Pull request by michio-rework**
Changes:
- removed react-is and webpack ( we don't need it directly in our deps I guess ) since we are not using them. 
- moved eslint-import-resolver-node, cli-select, commander

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [x] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,15 +44,16 @@
         "react-dom": "^17.0.2",
         "react-helmet": "^6.1.0",
         "react-i18next": "^11.15.3",
-        "react-is": "^17.0.2",
```
Comment (ali-hosseini-fs) [inline package.json:50]: dont we need this one anymore?

File: package.json [hunk 2]
```diff
@@ -47,15 +44,16 @@
...
         "react-paginate": "^8.1.0",
         "react-perfect-scrollbar": "^1.5.8",
         "react-ticker": "^1.3.1",
         "use-query-params": "^1.2.3",
-        "web-push-notifications": "^3.24.0",
-        "webpack": "^5.65.0"
```
Comment (ali-hosseini-fs) [inline package.json:56]: we need to check that removing this does not affect anything!

File: package.json (hunk 1 above)
Comment (michio-rework) [reply package.json:50]: I searched the whole project, it's not used anywhere. should we keep it?

File: package.json (hunk 2 above)
Comment (michio-rework) [reply package.json:56]: I checked around as far as I could, everything was fine, how about we add it in devDeps instead? 
definitely we don't need webpack in deps.

Comment (ali-hosseini-fs) [reply package.json:56]: we can pass it to qa to test it completely
if they dont find any issues then we can proceed with it

File: package.json (hunk 1 above)
Comment (ali-hosseini-fs) [reply package.json:50]: no, its ok,
lets remove it

File: package.json (hunk 2 above)
Comment (michio-rework) [reply package.json:56]: cool, will request their help on this

**Review by ali-hosseini-fs — APPROVED** [review]
LGTM

**Review by habib-deriv — DISMISSED** [review]

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request merged**